src/models/hooks/activityReportObjectiveFile.js

```diff
@@ -1,65 +1,10 @@
-const { getSingularOrPluralData } = require('../helpers/hookMetadata');
+/* eslint-disable import/prefer-default-export */
 const { cleanupOrphanFiles } = require('../helpers/orphanCleanupHelper');
 
-const recalculateOnAR = async (sequelize, instance, options) => {
-  // check to see if objectiveId or objectiveIds is validly defined
-  // when defined a more efficient search can be used
-  const objectiveIds = getSingularOrPluralData(options, 'objectiveId', 'objectiveIds');
-
-  let fileOnReport;
-  // by using the passed in objectives we can use a more performant version of the query
-  if (objectiveIds !== undefined
```

Comment:
maybe this was moved but its in the aro stuff just want to double check it should be moved.

Reply:
It's not moved, all it was doing was setting "onAr" on ObjectiveFiles, which is dropped